src/vdaf/prio3_instances.rs

```diff
@@ -0,0 +1,224 @@
+//! Implementation of the Prio3 instances from [[draft-irtf-cfrg-vdaf-01]].
```

Comment:
I think it's best to minimize code movement with this change. Instead of creating a new module, I would prefer to see [cfg(internal_prg)] tags on everything that you want to be able to exclude.

Reply:
Okay, I will undo and resubmit.